packages/docs/components/tab.md

```diff
@@ -25,16 +22,309 @@ links:
 
 ## Anatomy
 
+
+<Anatomy img="/images/anatomy-tab.svg" />
+
+## Behavior
+
 <Description>
 
-<span class="is-fpo">Descriptive content around **tab anatomy** should go here.</span>
+Users interact with Tab as they would a button, the main difference is the outcome. By default, the first tab from the left is active and the associated content displayed in the tab panel. Upon selecting a different tab, the tab indicator appears on the selected tab. The tab panel will then update to show the new content. 
+
+Additionally, the tab element is keyboard-navigable (See [Accessibility: Keyboard Support](#keyboard-support))
 
 </Description>
 
-<Anatomy img="/images/anatomy-tab.svg" />
+<Visual layout="wide" content="full" fade>
+  <template>
+    <header>
+      <h1>Terrestrial Planets</h1>
+      <p>Terrestrial planets are planets that are composed primarily of silicate rocks or metals.</p>
```

Comment:
Would love to hear about how this content is decided! This is great TIL content 😄